data/src/main/java/ca/bc/gov/data/local/entity/PatientEntity.kt

```diff
@@ -24,7 +25,11 @@ data class PatientEntity(
     @ColumnInfo(name = "time_stamp")
     val timeStamp: Instant = Instant.now(),
     @ColumnInfo(name = "patient_order", defaultValue = Long.MAX_VALUE.toString())
-    val patientOrder: Long
+    val patientOrder: Long,
+    @ColumnInfo(name = "authentication_status")
+    val authenticationStatus: AuthenticationStatus = AuthenticationStatus.NON_AUTHENTICATED,
+    @ColumnInfo(name = "unique_name")
```

Comment:
unique_name column is not required as we need to check for name and dob uniqueness only during insert and fetch